lisa/tools/lscpu.py

```diff
@@ -52,8 +52,13 @@ def __repr__(self) -> str:
         return self.__str__()
 
 
-ARCH_X86_64 = "x86_64"
-ARCH_AARCH64 = "aarch64"
+ArchitectureNames = {
+    "x86_64": CpuArchitecture.X64,
+    "aarch64": CpuArchitecture.ARM64,
+    "amd64": CpuArchitecture.X64,
+    "arm64": CpuArchitecture.ARM64,
+    "i386": CpuArchitecture.I386,
```

Comment:
Should it map to X64?

Reply:
I'm using it in my next PR that's based on this branch, it doesn't hurt to add it while I'm touching this code anyway.
This is my effort to break up reviews into smaller chunks 😁

Reply:
removed for now

Reply:
If it will be used, it's ok to add it. But if it's just for possible backward compatibility, it doesn't need. It's a long list for old architectures.